terraform/datadog/datadog.tf

```diff
@@ -0,0 +1,29 @@
+variable "datadog_api_key" {}
+variable "datadog_app_key" {}
+variable "env" {}
+
+provider "datadog" {
+    api_key = "${var.datadog_api_key}"
+    app_key = "${var.datadog_app_key}"
+}
+
+resource "datadog_monitor" "router" {
+  name = "${format("%s router hosts", var.env)}"
+  type = "service check"
+  message = "Missing router hosts in environment {{host.environment}}. Notify: @the-multi-cloud-paas-team@digital.cabinet-office.gov.uk"
```

Comment:
This is datadog markup. When alert goes off, {{host.environment}} gets replaced by the value of the 'environment' tag of the host that triggered alert. `Notify` directive specifies where to send alert to.

Edit: looking at alerts received, {{host.environment}} gets replaced into ''. Nevertheless, the email mentions environment name explicitly, so {{host.environment}} can be removed.